### PR TITLE
Update Readme and Stack Example

### PR DESCRIPTION
**Pull request by elrumordelaluz**
- Update README with new example code for Stacked Notifications and disclaimers.
- Remove `immutable.js` since it's currently not used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,23 +36,35 @@ Notification stack:
...
-state = {
-  notifications: OrderedSet().add({
-    message: 'Notification message'
-    key: 'some UID'
-  })
+
+// (...) Inside your Component
+
+constructor () {
+  suoper();
```
Comment (pburtchaell) [inline README.md]: Should be `super()`

**Comment by pburtchaell** [comment]
Thanks for adding the docs. Is there a particular reason why you wanted to remove Immutable from the example? I know it isn't used, but it makes it a bit more concise?

File: README.md (hunk 1 above)
Comment (elrumordelaluz) [inline README.md]: _ops!_ sorry for that, will fix it.

**Comment by elrumordelaluz** [comment]
>  Is there a particular reason why you wanted to remove Immutable from the example?

As I said in #63 
> I tryied to let the example as it is (using OrderedSetfrom Immutable.js) but didn't find a way to reference each generated Notification Component inside its own onClick prop, so I changed the example with a raw Object, referenced with an unique id based on Date.now() (with the same advice comment).

I didn't found a way to control that with ìmmutable.js`, so I rewrite with `Object.assign()`. 
Since there were as an unused package, I decided to remove it.

I know using `immutable.js` could be better. Please let me know if you think is possible to perform the example with this lib or what in another way.

**Comment by elrumordelaluz** [comment]
@pburtchaell Sorry for the delay on take this issue, week so busy. 

> why you wanted to remove Immutable

Just remade the example using `immutable`.

**Comment by pburtchaell** [comment]
No worries. It has been a busy week. I’ll review this over the weekend. Thanks!

**Comment by pburtchaell** [comment]
Thanks @elrumordelaluz. 

Also, I find immutableJS isn't necessarily better depending on the circumstance of the application. I like how simple the API is, so I figured it (being it is simple) would be best for the sake of the example.

**Comment by elrumordelaluz** [comment]
@pburtchaell absolutely agree.

**Pull request merged**